app.js

```diff
@@ -47,6 +49,10 @@ var Product = function(name, filePath, description){
 
 //-------- Functions ---------
 
+// Pushes DOM references to arrays

```

Comment:
this is the perfect way to prepare yourself for adding multiple images to the page. The fix you need to make is that right now, the way we are building the page is manually in the html. Instead, try building the images all the way from scratch. You could then build 3 images, 5 images, 2 images etc.